eventservice/api/tags.json

```diff
@@ -1 +1 @@
", "homecoming", "Honors LLC", "Hopkins Hall", "Housing Sign Up", "human rights", "humanities", "iceskating", "iGuide", "iGuides", "Ike", "Ike North", "Ike South", "Ikenberry Commons", "Illini Guides", "Illini Union", "illinois", "Illinois Residential Experience", "Illinois Residential Leadership Experience", "implicit bias", "information session", "innovation", "innovation expo", "Innovation LLC", "international", "Interpersonal Competence", "Intersections", "investiture", "ISR", "japan", "japanese culture", "korea", "labor", "LAR", "Latino Student Association", "latino studies", "Leadership Suite", "LEADS", "learning", "lecture", "Leonard Hall", "live music", "LSA", "lunch", "lunch period", "lunchtime", "Ma'at", "Mariama", "MAs", "McKinley Health Center", "Meal Plan", "Men of Impact", "mentoring", "mhrir", "microaggressions", "mindfulness", "Moms' Day Brunch", "Move In", "MTD", "Multicultural Advocates", "museum", "music", "music practice rooms", "musical theatre", "national career development month", "National Residence Hall Honorary", "natural history", "nature", "networking", "no cost", "NRHH", "Nugent Hall", "nutrition", "Oglesby Hall", "oncology", "open house", "Orchard Downs", "outdoors", "outreach", "painting", "panel discussion", "PAR", "Parking", "Penn Station", "performance", "performance art", "persistence", "ph.d. advisory committee", "physics", "political science", "politics", "presentation", "prevention", "professional development", "proficiency exam", "public meeting", "publishing", "puzzles", "Queer Housing Coalition", "queer identity", "race", "RAs", "reading group", "recreation", "regional dynamics in northeast asia", "relaxation", "research", "research abroad", "research strategies", "research workshop", "Residence Hall Libraries", "Resident Advisors", "risk management", "rodeo", "roommate", "RSO", "run/walk", "running", "Salongo", "Saunders Hall", "scholarship", "scholarships", "school nutrition", "science", "science booth at farmers market", "Scott Hall", "SDRP", "SDRP", "self-care", "service", "Sheldon Hall", "Snyder Hall", "social", "social identity", "social justice", "Social Justice Exploration", "spirituality", "sports history", "startups", "stem", "stress management", "student activism", "student support", "students", "study abroad", "summer activities", "support", "sustainability", "Sustainability", "Taft Hall", "teaching & learning", "theater", "thesis defense", "town hall meeting", "Townsend Hall", "Trelease Hall", "TVD", "undergraduate research", "undergraduate research symposium", "undergraduate research week", "undergraduates and recent alumni", "Unit One", "University Housing", "Urbana North", "Urbana South", "urs", "urw", "VanDoren Hall", "Variety Show", "volunteering", "walking", "Wardall Hall", "Wassaja Hall", "webcast", "webinar", "welcome", "wellness", "Wellness", "Weston Exploration", "Weston Hall", "WEX", "WIMSE", "wine tasting", "women's history", "workshop", "Xfinity", "yoga", "Aquatics", "Intramural"]
```

Comment:
Please insert `Aquatics` and `Intramural` in alphabetical order in this list as discussed earlier today. Thanks.